examples/todo.py

```diff
@@ -28,7 +28,7 @@ def add_task(*ags, **kws):
 
     # add the task element to the page as new node in the list by cloning from a
     # template
-    task_html = task_template.clone(task_id, to=task_list)
```

Comment:
I couldn't find a definition for this `clone` function anywhere... Not even on the docs!
Do you know exactly what this function is doing and what's this `to` argument?
I thought it was adding items to the list programmatically, but I just tested locally and it's not.

Reply:
Yes, you can find the definition as a method of the Element class in https://github.com/pyscript/pyscript/blob/main/pyscriptjs/src/pyscript.py
on line 91

Reply:
Yeah! Sorry I missed that!